adapters/repos/db/vector/hnsw/vectors_for_test.go

```diff
@@ -28,6 +28,6 @@ var testVectors = [][]float32{
 	{0.8, 0.7},
 }
 
-func testVectorForID(ctx context.Context, id int32) ([]float32, error) {
+func testVectorForID(ctx context.Context, id int64) ([]float32, error) {
```

Comment:
see above, all node ids should be represented by `uint64`, removing the need for the conversions.